apps/pyth-entropy/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "pyth-entropy",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
```

Comment:
Thoughts on adding forge create and node fulfillMint, as something like:

"deploy": "forge..."
"fulfill": 
"request"..

Reply:
Good idea! I think this could help in other guides, but because the `forge create` command requires so many arguments, it wouldn't be as user friendly to make them pass in their env variables, as opposed to loading it from `.env`

Users would still have to do something like this otherwise:
`pnpm run deploy -- --PRIVATE_KEY=your_private_key --RPC_URL=your_rpc_url --ENTROPY_ADDRESS=your_entropy_address --PROVIDER_ADDRESS=your_provider_address`